src/Layer/GeometryLayer.js

```diff
@@ -193,6 +193,14 @@ class GeometryLayer extends Layer {
         this.object3d.traverse(disposeMesh);
     }
 
+    /**
+     * Deprecated function. Use dispose()
+     */
+    delete() {
+        console.warn('`GeometryLayer.delete` method is deprecated. Please use `GeometryLayer.dispose` instead.');
+        this.dispose();
+    }
+
```

Comment:
Deprecated function `delete` could be only in `Layer` to avoid that it is only in each subclass of `Layer`?